### PR TITLE
Fix errors during Database Rotation

### PR DESCRIPTION
**Pull request by KyleAure**
failure_group-f6233dd1-21a0-4f6f-b70b-806aac880b47
W CWWJP9991W: [eclipselink.server] Failed to find MBean Server: null or empty List returned from MBeanServerFactory.findMBeanServer(null)

failure_group-c3b9f95b-2141-494c-8424-d1b759733dca
PostgreSQL
Caused by: java.lang.ClassNotFoundException: org.postgresql.util.PGobject

failure_group-cc166913-e9d4-4038-a13d-561416c9cf10
PostgreSQL
Caused by: java.lang.ClassNotFoundException: org.postgresql.util.PGobject

### CODE REVIEW

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
PosgreSQL SOE: https://libh-proxy1.fyre.ibm.com/cognitive-dev/pipelineAnalysis.html?pipelineId=460faad4-5171-4af6-b557-9812ec6dac92

Target locations of links might be accessible only to IBM employees.

File: dev/com.ibm.ws.jpa.tests.beanvalidation_fat.common/fat/src/com/ibm/ws/jpa/tests/beanvalidation/tests/BeanValidation20_EJB.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa.tests.beanvalidation_fat.common/fat/src/com/ibm/ws/jpa/tests/beanvalidation/tests/BeanValidation20_EJB.java:4]: Copyright should add 2024

**Review by tkburroughs — CHANGES_REQUESTED** [review]
Copyright in all files should end in 2024

**Review by Riva-Tholoor-Philip — DISMISSED** [review]
Changes Approved

**Comment by KyleAure** [comment]
RTC Build: Failed with known defects
Pipeline: Passed successfully
PostgreSQL: Failed with issues previously overshadowed by those issues fixed by the PR.

**Review by tkburroughs — APPROVED** [review]

**Comment by tkburroughs** [comment]
#libby

**Pull request merged**